site/content/usage/12-cloudwatch-cluster-logging.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Enabling CloudWatch logging"
+weight: 120
+url: usage/cloudwatch-cluster-logging
+---
+
+[eksdocs]: https://docs.aws.amazon.com/eks/latest/userguide/control-plane-logs.html
+
+## Enabling CloudWatch logging
+
+[CloudWatch logging][eksdocs] for EKS control plane is not enabled by default due to data
+ingestion and storage costs.
+
+To enable control plane logging when cluster is created, you will need to define **`cloudWatch.clusterLogging.enableTypes`** setting in your `ClusterConfig` (see below for examples).
+
+So if you have a config file with correct **`cloudWatch.clusterLogging.enableTypes`**
+setting, you can create a cluster with `eksctl create cluster --config-file=<path>`.
+
+If you have created a cluster already, you can use `eksctl utils update-cluster-logging`.
+
+> **NOTE**: this command runs in plan mode by default, you will need to specify `--approve` flag to
+> apply the changes to your cluster.
+
+If you are using a config file, run:
+```
+eksctl utils update-cluster-logging --config-file=<path>
+```
+
+Alternatively, you can use CLI flags.
+
+To enable all types of logs, run:
+```
+eksctl utils update-cluster-logging
+```
+
+To enable `audit` logs, run:
+```
+eksctl utils update-cluster-logging --audit
```

Comment:
This didn't get updated, I've opened https://github.com/weaveworks/eksctl/issues/1090.